GVFS/GVFS.FunctionalTests/Tests/EnlistmentPerTestCase/RepairTests.cs

```diff
@@ -13,6 +13,16 @@ namespace GVFS.FunctionalTests.Tests.EnlistmentPerTestCase
     [Category(Categories.MacTODO.M4)]
     public class RepairTests : TestsWithEnlistmentPerTestCase
     {
+        [TestCase]
+        public void NoFixesNeeded()
+        {
+            this.Enlistment.UnmountGVFS();
+

```

Comment:
Another style nit: I think there are an excessive number of blank lines in these tests. It's hard to read code when each line of code is followed by a blank line :-)